GVRf/Framework/backend_oculus/src/main/java/org/gearvrf/OvrMonoscopicViewManager.java

```diff
@@ -15,10 +15,17 @@
 
 package org.gearvrf;
 
+import org.gearvrf.utility.Log;
 import org.gearvrf.utility.VrAppSettings;
 
 import android.app.Activity;
 import android.util.DisplayMetrics;
+import android.view.SurfaceHolder;
+import android.view.SurfaceView;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
 
 /*
  * This is the most important part of gvrf.
```

Comment:
Need to add Vulkan-specific comments. 

_Maybe_ we need to rethink Vulkan design because this comment block says that there's nothing GL-specific in the constructor, but we do Vulkan init in the constructor.

**Possibilities**:
1. the original comment is wrong (there _is_ some GL-specific code in constructor and adding Vulkan code is OK): **need to rewrite the comment**
2. the oringinal _design_ is wrong (works with GL, but Vulkan is _so_ different that we need to rethink the whole design): **rethink the design, write new documentation, redo GL**
3. Vulkan implementation is not following the original design (design holds, but Vulkan as-is is built right across it): **redo Vulkan**